SAGE/Games/Kane's Wrath/Includes/AudioSettings.xml

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<AssetDefinition xmlns="uri:thundermods.net:SAGE:WrathEdXML:AssetDefinition">
+
+	<Asset
+		id="MicrophoneSettings">
+		<EntryInheritance
+			id="base"
+			AssetType="BaseAssetType" />
+		<Entry
+			id="fill"
+			AssetType="Byte"
+			IsVoid="true" />
```

Comment:
Without it, the attributes would be offset, the values wouldn't match the original version "DATA:sounds/audiosettings.xml". So it would either crash or I would notice the correct value in a different entry. Fundamentally I don't know why it's like that and if it is supposed to be something. There is something similar with Modules/WeaponSetUpdate WeaponSlot_Hardpoint